includes/wikia/api/ArticlesApiController.class.php

```diff
@@ -983,7 +983,12 @@ public function getAsJson() {
 			}
 
 			if ( $redirect !== 'no' && $article->getPage()->isRedirect() ) {
-				$article = Article::newFromTitle( $article->getPage()->followRedirect(), RequestContext::getMain() );
+				// this will be equal to false if redirect is not local
```

Comment:
Are you sure? As I glanced at the code it seems it can also return a string with an URL. How about changing condition to:
```php
if ( $followRedirect && !is_string( $followRedirect ) ) {
    $article = Article::newFromTitle( $followRedirect, RequestContext::getMain() );
}
```
or we can go with `instance of` but I recall it isn't recommended in terms of performance.